dp/cloud/python/magma/configuration_controller/response_processor/response_db_processor.py

```diff
@@ -45,6 +46,7 @@ def __init__(
         self.request_states_map = {}
         self.fluentd_client = fluentd_client
 
+    @SAS_RESPONSE_PROCESSING_TIME.time()
```

Comment:
Not sure about this, I though we were interested in time between sending request to SAS and receiving response, but here it is just internal response processing time.

Reply:
true, i just put it there as just another metric. to get more granularity in case if it affects the overall request processing time